html/donate/aa.html

```diff
@@ -201,28 +201,26 @@ <h3>Help us to fund the Open Food Facts 2020 budget!</h3>
             paypalExpress="true"
           ></script>
           <iframe
+            title="Donate via Donorbox"
             allowpaymentrequest=""
-            frameborder="0"
             height="900px"
-            name="donorbox"
-            scrolling="no"
-            seamless="seamless"
+            name="donorbox"seamless="seamless"
```

Comment:
that one is missing a space, but it seems it did not affect other  translations

those files are generated by crowdin from the en.html file I think

Reply:
I have to admit I didn't check all files individually after regex replace. 🙂 Fixed this, thanks!